backend/apps/volontulo/views/api.py

```diff
@@ -323,3 +323,23 @@ def post(self, request):  # pylint:disable=no-self-use,unused-argument
             send_copy_to_admin=False,
         )
         return Response({}, status.HTTP_201_CREATED)
+
+
+class PasswordChangeView(APIView):
+    """Password change view."""
+    permission_classes = (IsAuthenticated, )
+    authentication_classes = (CsrfExemptSessionAuthentication,)
```

Comment:
IMO for these classes we don't need commas (and space in the first one).

Reply:
It is a tuple so the parenthesis and commas are required. I removed the space after comma.